LightAPRS-hab/LightAPRS-hab.ino

```diff
@@ -323,9 +318,7 @@ void updateTelemetry() {
   telemetry_buff[49] = ' ';
   sprintf(telemetry_buff + 50, "%02d", gps.satellites.isValid() ? (int)gps.satellites.value() : 0);
   telemetry_buff[52] = 'S';
-  telemetry_buff[53] = ' ';
-  sprintf(telemetry_buff + 54, "%s", comment);
```

Comment:
I think removing the sprintf() call might change the telemetry_buff char array to not end with a null character '\0'. So you may need to manually append the null character to the array in order to not cause any errors.